asammdf/gui/ui/file_widget.ui

```diff
@@ -20,11 +20,11 @@
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>2</number>
      </property>
      <widget class="QWidget" name="channels_tab">
       <attribute name="icon">
-       <iconset resource="resource.qrc">
+       <iconset>
```

Comment:
I don't think the icons will be properly displayed without the resource file. Di you set it up in QtDesigner?

Reply:
was shown in the settings, but not correctly, resolved with reopening of the resource file